mlrun/utils/helpers.py

```diff
@@ -536,6 +537,11 @@ def splitter(verb, text):
     return match
 
 
+def enrich_owner_label(labels: dict, owner: str = None):
```

Comment:
Let's generalize it. We can use Enum for the RunLabels. The helper will enrich the given label something like:
```
def enrich_labels(labels: dict, labels_to_enrich: list = None):
       run_labels = {RunLabels.owner: ....., .....}
       labels_to_enrich = labels_to_enrich or RunLabels.all()
       for label_to_enrich in labels_to_enrich:
             ....
```
Note owner and v3io_user is duplication we want to remove someday

Reply:
It's a good idea, but enrichment is usually for default values. 
I didn't find any other default labels other than `owner` and `v3io_user`. Do you know of any others?

Also, if we want to remove `v3io_user`, shouldn't it be written somewhere?

Reply:
> Also, if we want to remove v3io_user, shouldn't it be written somewhere?

There is a TODO somewhere.
I'm not aware of more run labels ATM but it shouldn't be an issue